### DISCUSSION TITLE
Cant configure Synpress on my project ( base url problem and tsconfig.json not found issue)

### DISCUSSION THREAD

**Discussion by Belf33** [Synthetixio/synpress#524] [General]
Hi to all. 
Trying to find a decision more than one week, but did find one.

So I do not understand how to change base URL for app run. 
I found only one way, change it in node_modules is synpress ligrary itself, but that's not right I think

second issue, I go though synpress readme 20 times, trying all that I saw there, but I still see issues when I try to run on my project. Tsconfig.json still couldn't find.  

here my code https://github.com/Belf33/Synpress_web3_UI_framework_mvp/tree/refactoring

Can someone tells me what I do not see ?

![image](https://user-images.githubusercontent.com/20465760/193464206-0ee219de-8bf9-4edf-b7ae-2b7fdc2cbe1e.png)

**Comment by Belf33** [comment]
Update: Sory, was my fault. Found the way how to resolve issue

**Reply by drptbl** [reply]
This may be also helpful:
https://github.com/drptbl/synpress-setup-example

**Reply by emmabubu** [reply]
Could you post your solution here? I encountered the same issue.
